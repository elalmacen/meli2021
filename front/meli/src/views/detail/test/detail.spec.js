import React from 'react';
import {shallow} from 'enzyme';
import View from '../detail';

describe('test detail/detail view', () => {
    test('renders without crashing', () => {
        const wrapper = shallow(<View />);
        expect(wrapper).toMatchSnapshot();
    });    
});
