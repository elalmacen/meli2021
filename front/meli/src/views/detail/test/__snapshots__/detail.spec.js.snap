// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test detail/detail view renders without crashing 1`] = `
<Fragment>
  <Product
    categories={Array []}
    notFound={false}
    product={Array []}
  />
</Fragment>
`;
